packages/auth/src/AuthService.ts

```diff
@@ -68,4 +68,21 @@ export class AuthService {
   public extract(): KeycloakInstance {
     return this.auth;
   }
+
+  /**
+   * Check it the user has a specified realm role
+   */
+  public hasRealmRole(role: string): boolean {
+    return this.auth.hasRealmRole(role);
+  }
+
+  /**
+   * Return the users realm level roles
+   */
+  public getRealmRoles(): string[] {
+    if (this.auth && this.auth.realmAccess && this.auth.realmAccess.roles) {
```

Comment:
[Optional] strange to check this.auth where we do not check that in other cases.

Reply:
Good catch was just an overlook on my part when trying to get it build